frontend/packages/core/src/card.tsx

```diff
@@ -36,23 +38,71 @@ const StyledCardHeaderContainer = styled.div({
   background: "#EBEDFB",
 });
 
-interface CardHeaderProps extends Pick<MuiCardHeaderProps, "avatar" | "title"> {
+const StyledCardHeader = styled(Grid)({
+  padding: "16px",
+  minHeight: "72px",
+  ".MuiGrid-item": {
+    padding: "0px 8px 0px 8px",
+  },
+});
+
+// TODO: use material ui avatar component and implement figma design
+const StyledCardHeaderAvatar = styled.div({
+  width: "32px",
+  height: "32px",
+  marginRight: "16px",
+});
+
+const StyledDivider = styled(Divider)({
+  color: "rgba(13, 16, 48, 0.38)",
+  height: "36px",
+  alignSelf: "center",
+});
+
+const StyledGridItem = styled(Grid)({
+  textAlign: "center",
+});
+
+export interface CardHeaderSummaryProps {
+  title: React.ReactNode;
+  subheader?: string;
+}
+
+interface CardHeaderProps {
+  avatar: React.ReactNode;
   children?: React.ReactNode;
+  summary?: CardHeaderSummaryProps[];
+  title: React.ReactNode;
```

Comment:
nit:
```suggestion
  title: React.ReactElement<Typography>;
```

Reply:
I tried it but it didn't enforce the type when I used another component. Also tried `React.ReactElement<Typography> | React.ReactElement<Typography>[]`; , a pattern we use in the codebase but that didn't enforce the type either 🤔 still looking into, maybe it's a typescript bug

Reply:
did some more local testing with other components that type React children (https://github.com/lyft/clutch/search?q=React.ReactElement), and type declarations like these don't enforce the type of the children

it seems like it isn't currently possible to type-check React children (https://stackoverflow.com/questions/59381291/react-typescript-only-allow-children-of-specific-type, https://github.com/Microsoft/TypeScript/issues/14729)

however, i pushed up the change as I think there's still value in documenting what elements _should_ be passed in